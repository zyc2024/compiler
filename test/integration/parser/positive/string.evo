

char[] addBubbles (char[] s1) {
    return s1 + "bub bl e's";
}