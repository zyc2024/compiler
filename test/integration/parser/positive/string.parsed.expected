(
    ()
    (
        (addBubbles ((s1 ([] char) )) (([] char)) ( 
            (return (+ s1 "bub bl e's" )
        )   )
    )
))